packages/yoco/svg/eye_slashed.svg

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg width="800px" height="800px" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Those "800px" look a bit sus. Not sure if it has an actual effect, but I would change both to "24px" (as for most of yoco icons) just to be safe.